player/js/modules/canvas_worker.js

```diff
@@ -0,0 +1,80 @@
+import lottie from './canvas';
+import AnimationItem from '../animation/AnimationItem';
+import CanvasRenderer from '../renderers/CanvasRenderer';
+
+// Monkey patch some methods to work correctly with Worker
+AnimationItem.prototype.resize = function (width, height) {
+  this.renderer.updateContainerSize(width, height);
+};
+
+CanvasRenderer.prototype.updateContainerSize = function (width, height) {
```

Comment:
> this could be moved to `player\js\renderers\CanvasRendererBase.js` lines 208 and 209
> 
> ```js
>     elementWidth = width || this.canvasContext.canvas.width * this.renderConfig.dpr;
>     elementHeight = height || this.canvasContext.canvas.height * this.renderConfig.dpr;
> ```

Yeah, sounds good, the monkey patching was a temporary decision before getting in touch with you. Perfect that we can extend it right here